### ISSUE TITLE
Search input issues (WCAG 1.3.1 & 4.1.3) [QAC]

### ISSUE THREAD

**Issue by ftourangeau** [MeD-DMC/Adapt-accessibilityfixes#205]
**COURSE:**
https://esdc-edsc-adapt.canadacentral.cloudapp.azure.com/preview/5dc446fd27f08836496b6b9e/63a32079b75e454dafed1e1d/
*******************************************************************************
**Page:** 
Main Hamburger Menu

**Description:**
There is a visual label with the "Search" input field, however, they are not associated.

Each input field, select, textarea, progress, meter, and output element must have a name available to the accessibility API.

Solution:
Please use a <label> element with a for attribute. The "for=" attribute of the label must match the "id=" attribute of the input tag.
![image](https://user-images.githubusercontent.com/93537804/229625868-dbed86eb-5e8f-4612-854e-f5de43c0ccac.png)

This issue triggers also SC 4.1.2 Name, Role, Value [Level A].
*******************************************************************************
**Page:** 
Main Hamburger Menu

**Description:**
There is no notification for AT users that the search result is updated.

Please notify AT users each time the search result has changed. For example:
   • 5 results are available
   • No results found
![image](https://user-images.githubusercontent.com/93537804/229626021-a49b3316-9295-4e6c-a328-73db54db8aa2.png)
*******************************************************************************
